hbase-server/src/main/java/org/apache/hadoop/hbase/util/DirectMemoryUtils.java

```diff
@@ -77,36 +76,13 @@ public class DirectMemoryUtils {
     HAS_MEMORY_USED_ATTRIBUTE = a != null;
   }
 
-  /**
-   * @return the setting of -XX:MaxDirectMemorySize as a long. Returns 0 if -XX:MaxDirectMemorySize
-   *         is not set.
-   */
+  /** Returns the direct memory limit of the current progress */
   public static long getDirectMemorySize() {
```

Comment:
Will this method be called by multiple threads? Let's initialize it while loading the class?